### PR TITLE
Apollos 1.6.0

### PR DESCRIPTION
**Pull request by asleepingpanda**
Updates to Apollos 1.6.0

Biggest changes coming in the form of organization of the Feature package. All references to `createGlobalId` have been moved from the Feature dataSource to individual Feature Resolvers.

There is also an app branch to go along with this update, but we need to insure that it is backwards compatible before launching.

### CODE REVIEW

**Comment by asleepingpanda** [comment]
@redreceipt would love to get your eyes on this, too, to make sure it all looks sound!

File: src/data/action-algorithms/data-source.js [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+import ApollosConfig from '@apollosproject/config'
+
+const { ROCK_MAPPINGS, ROCK } = ApollosConfig
+const { createImageUrlFromGuid } = Utils
+
+export default class ActionAlgorithm extends coreActionAlgorithm {
+  expanded = true
+
+  /** Base Attrs and Methods from the Core DataSource */
+  baseAlgorithms = this.ACTION_ALGORITHIMS;
```
Comment (redreceipt) [inline src/data/action-algorithms/data-source.js:19]:
```suggestion
  baseAlgorithms = this.ACTION_ALGORITHMS;
```

we spelled it correctly this time 😬

File: src/data/action-algorithms/data-source.js [hunk 2]
```diff
@@ -0,0 +1,261 @@
...
+
+  // Names of Action Algoritms mapping to the functions that create the actions.
+  ACTION_ALGORITHIMS = Object.entries({
```
Comment (redreceipt) [inline src/data/action-algorithms/data-source.js:22]:
```suggestion
  ACTION_ALGORITHMS = Object.entries({
```

File: src/data/features/data-source.js [hunk 3]
```diff
@@ -293,25 +288,21 @@ export default class Feature extends coreFeatures.dataSource {
     }
 
     async createAvatarListFeature({ algorithms, primaryAction, isCard }) {
-        const people = () => this.runAlgorithms({ algorithms });
+        const people = this.runAlgorithms({ algorithms });
```
Comment (redreceipt) [inline src/data/features/data-source.js:291]:
```suggestion
        const people = ActionAlgorithm.runAlgorithms({ algorithms });
```

**Comment by redreceipt** [comment]
Actually, I think my comments may be for 1.7, I can't remember where that was added in. You might be ok as is.

**Comment by asleepingpanda** [comment]
@rkdavidson and/or @jordanwade could I get a review on this so that we can push out another TestFlight??

**Review by jordanwade — APPROVED** [review]

**Pull request merged**